.github/workflows/release.yml

```diff
@@ -38,7 +38,6 @@ jobs:
           yarn workspace @lightningjs/ui-components run semantic-release --no-ci -e semantic-release-monorepo 
           yarn workspace @lightningjs/ui-components-test-utils run semantic-release --no-ci -e semantic-release-monorepo
         env:
-          GH_TOKEN: ${{secrets.GH_TOKEN}}
```

Comment:
we dont need to replace this with another token?

Reply:
From what I am reading a new token will be added to the workflow each time it runs. So I don't believe we will need to add it. https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
Oh, actually. I need to reference it through secrets. Will make that update.